README.md

```diff
@@ -39,6 +39,22 @@ If you don't have openshift running
 oc cluster up
 ```
 
+Create a new target project
+```bash
+oc new-project <target-project>
+```
+
+Now set the `NAMESPACE` env var to point at the `target-project`
+```bash
+export NAMESPACE=<target-project>
+```
+
+Also set `KUBERNETES_CONFIG` to point to your `.kube/config`
+```bash
+export KUBERNETES_CONFIG=$HOME/.kube/config
```

Comment:
There is already a cli flag created to set this value: https://github.com/aerogear/mobile-client-service/blob/master/cmd/api-server/main.go#L105. We should either use it as well, or remove it completely and just use the env var.

Reply:
The operator sdk needs `KUBERNETES_CONFIG` set but I can do
```.go
os.Setenv("KUBERNETES_CONFIG", kubeconfig)
```
using the flag value if its defined.

What is the syntax to pass namespace through `make serve`? I have always been using the env var